backend/controller/sql/schema/001_init.sql

```diff
@@ -263,6 +263,27 @@ CREATE TABLE cron_jobs
 CREATE INDEX cron_jobs_executing_start_time_idx ON cron_jobs (start_time) WHERE state = 'executing';
 CREATE UNIQUE INDEX cron_jobs_key_idx ON cron_jobs (key);
 
+CREATE OR REPLACE FUNCTION create_cron_job(
```

Comment:
I can't decide if this is better or worse 😬 than symlinking the query files...WDYT? My gut feeling is that this is worse, but I'm not sure.

Reply:
Yeah... tbh I do prefer the symlinking. I actually consulted copilot for the first time ever when writing this because it was tricky to get the syntax just right... I also like how the symlinking would let us break up the shared queries into smaller scope directories. It's a little anxiety inducing envisioning this file filling up with every shared query under the sun. Maybe I'll ask the team during standup today to get other folks' opinions... the previous commit on this branch has the symlinking, so that makes it easy enough to show everyone

Reply:
I think let's just go with symlinking as we both agree.

Reply:
Sweet sounds good!